src/System.Composition.AttributedModel/src/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Resources;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Security;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyTrademark("")]
+[assembly: AssemblyCulture("")]
+[assembly: NeutralResourcesLanguage("en")]
+
+[assembly: ComVisible(false)]
+[assembly: SecurityTransparent]
```

Comment:
@stephentoub Transparency checks [have been disabled](https://github.com/dotnet/coreclr/commit/725d6a91fffdf4603c8a9c74e15cbc83ebf0e3d6) in .NET Core.  Adding System.Composition to .NET Core was actually blocked on this, as there was an issue where dynamically compiled code would run into security exceptions no matter what transparency attributes were applied.

However, I would suggest leaving the transparency attributes in this code base as is, to reduce the chance of breaking something on the other platforms this library supports.  I think Windows Phone 8/8.1 would by risky here.